lib/api/storage.ts

```diff
@@ -18,11 +20,15 @@ export class ShredderStorage {
   }
 
   static getRaw() {
-    return localStorage.getItem(`${ location.origin }-shredder`);
+    return localStorage.getItem(this.storageKey);
   }
 
   static getSerialized() {
-    return JSON.stringify(ShredderStorage.getRaw());
+    return JSON.stringify(this.getRaw());
+  }
+
+  static getParsed() {
+    return JSON.parse(this.getRaw());
```

Comment:
@ilyagelman Is it ok to use `this`? What's the reason for using the class name outside of the `constructor`?